### PR TITLE
Auto login using Openlogin session

### PR DESCRIPTION
**Pull request by tainguyentt**

### CODE REVIEW

File: src/handlers/Auth/OpenLoginHandler.js [hunk 1]
```diff
@@ -56,8 +56,8 @@ class OpenLoginHandler {
 
   async getActiveSession() {
     try {
+      log.info(this.openLoginInstance.state.store.getStore(), 'store')
```
Comment (tainguyentt) [inline src/handlers/Auth/OpenLoginHandler.js:59]: remove log

File: src/handlers/Auth/OpenLoginHandler.js [hunk 2]
```diff
@@ -81,6 +81,7 @@ class OpenLoginHandler {
       releaseLock()
       return this.openLoginInstance
     }
+    log.info(this.openLoginInstance.state.store.getStore())
```
Comment (tainguyentt) [inline src/handlers/Auth/OpenLoginHandler.js:84]: remove log

**Review by tainguyentt — COMMENTED** [review]
tested in Chrome, Firefox, Safari, Brave, and it worked for me.

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Review by himanshuchawla009 — CHANGES_REQUESTED** [review]

**Comment by himanshuchawla009** [comment]
https://user-images.githubusercontent.com/14052374/183077149-cc70f020-b336-463a-acaf-e3f677b4bf1f.mov

there is some lag in opening embed widget

**Comment by himanshuchawla009** [comment]
logout event is not propagating from wallet to embed.

- login with embed
- Open wallet via embed widget
- logout from wallet
- User still remains logged in dapp

File: src/config.js [hunk 3]
```diff
@@ -91,14 +91,46 @@ const redirectURI = `${baseUrl}/redirect`
...
+/**
+ * Checks whether a storage type is available or not
+ * For more info on how this works, please refer to MDN documentation
+ * https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API#Feature-detecting_localStorage
+ *
+ * @method storageAvailable
+ * @param {String} type the type of storage ('localStorage', 'sessionStorage')
+ * @returns {Boolean} a boolean indicating whether the specified storage is available or not
+ */
+export function storageAvailable(type) {
```
[the remark this answers is not in the document]
Comment (chaitanyapotti) [reply src/config.js:107]: we would create a circular dependency otherwise

**Comment by tainguyentt** [comment]
I did one round of testing across browsers (Chrome, Brave, Safari, Firefox) and didn't find any issue yet

**Review by himanshuchawla009 — APPROVED** [review]
lgtm

**Pull request merged**